### PR TITLE
Migrate config to YAML

### PR DESCRIPTION
**Pull request by indiv0**
Remove example python config file.
Update .gitignore to ignore the new configuration file.
Add pyyaml as a dependency.
Add an example YAML configuration file.
Modify config code to load the configuration from a YML file.
Update README to refer the the proper example config name.
Fix PEP 8 issues.

### CODE REVIEW

File: Pipfile [hunk 1]
```diff
@@ -7,6 +7,7 @@ aiohttp = ">=2.2.3,<2.3.0"
 async-timeout = ">=1.2.1,<1.3.0"
 chardet = ">=3.0.4,<3.1.0"
 multidict = ">=3.1.3,<3.2.0"
+pyyaml = ">=3.12,<3.13"
```
Comment (SunDwarf) [inline Pipfile]: `PyYAML` is outdated - `ruamel.yaml` is the better package https://pypi.python.org/pypi/ruamel.yaml

Comment (Fuyukai) [inline Pipfile]: PyYAML is outdated - [ruamel.yaml](https://pypi.python.org/pypi/ruamel.yaml) is better

File: run.py [hunk 2]
```diff
@@ -1,10 +1,22 @@
...
 def main():
     """Launches the bot"""
-    bot = Union()
+    # Load the configuration
+    with open("config.yml", 'r') as stream:
+        try:
+            config = load(stream)
+        except YAMLError as exc:
+            print(exc)
+            sys.exit(1)
```
Comment (Fuyukai) [inline run.py]: Perhaps make this a return instead

File: run.py [hunk 3]
```diff
@@ -1,10 +1,22 @@
...
+from yaml import load, YAMLError
+
 
 def main():
     """Launches the bot"""
-    bot = Union()
+    # Load the configuration
+    with open("config.yml", 'r') as stream:
+        try:
+            config = load(stream)
```
Comment (Fuyukai) [inline run.py]: Use `yaml.CLoader` if possible.

**Review by Fuyukai — CHANGES_REQUESTED** [review]

**Review by Fuyukai — APPROVED** [review]

**Pull request merged**